packages/phoenix-client/src/phoenix/client/resources/experiments/__init__.py

```diff
@@ -676,6 +670,52 @@ def sync_run_task(test_case: TestCase) -> Optional[ExperimentRun]:
 
         return ran_experiment
 
+    def _get_all_experiment_runs(
+        self,
+        *,
+        experiment_id: str,
+        page_size: int = 100,
```

Comment:
might eventually parameterize or let the server dictate here given it's hard to know the payload size. 100 could be a lot or a little.